tests/test_program.py

```diff
@@ -5,6 +5,7 @@
 from typing import Any, Dict, Iterable, List, MutableSet, Sequence
 
 import pytest
+from xir import ObservableFactor
```

Comment:
Our style convention is to write `xir.ObservableFactor` instead of just `ObservableFactor`.
```suggestion
```